src/app/zap-templates/zcl/application-launcher-cluster.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Application Launcher</name>
+    <code>0x050c</code>
+    <define>APPLICATION_LAUNCHER_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
+
+    <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST" type="OCTET_STRING" length="32" writable="false" optional="false">application launcher list</attribute>
```

Comment:
should this be named 'catalog list' to match the spec?

I am unclear from the spec type `[list-def][uint16]` what type this should be. Could you explain this a bit?

Reply:
As far as I know, list is currently unsupported, this is "temporary" type until list is added.